### ISSUE TITLE
New feature: customizable padding / element spacing 

### ISSUE THREAD

**Issue by Xeoncross** [marktext/marktext#902]
### Describe your feature request

Allow the user to specify how much padding is between each element in the document. By default the content does not have the right typographic balance. Allow the user to specify larger whitespace gaps.

![Screen Shot 2019-04-09 at 10 42 41 PM](https://user-images.githubusercontent.com/56460/55850252-34d20580-5b19-11e9-8c74-d549983ed386.png)

This feature will improve readability for many types of content. I assume elements are spaced with a simple padding or margin CSS rule.

**Comment by Jocs** [comment]
This feature should be implemented in the Add Custom theme, and you can add your favorite themes and modify the CSS styles. Including the spacing between paragraphs.

**Comment by Xeoncross** [comment]
I disagree based on the existing options already available in the preferences. 

![Screen Shot 2019-04-10 at 8 53 45 AM](https://user-images.githubusercontent.com/56460/55884747-3843ac00-5b6e-11e9-8c87-025b0a888d60.png)

I do not want a new theme, I simply want the line-height + padding to keep vertical rhythm. The default looks like `0.5 * 2 = 1em` while the default line-height is `1.6em`.

**Comment by Jocs** [comment]
>I do not want a new theme, I simply want the line-height + padding to keep vertical rhythm. The default looks like 0.5 * 2 = 1em while the default line-height is 1.6em.

Yes, But another problem is not that the spacing of each paragraph is the same, such as the title and title, the spacing between the title and the paragraph should be different, although it is now the same in Mark Text. We are also thinking about this issue, and we will adjust the spacing between paragraphs according to different paragraph types.

If you feel unsatisfied at that time(after we adjust the space between paragraph), we recommend that you download the source code, modify the original theme style, and then package it yourself, or add a new theme.

Now I label this issue as `improvement`.
